### DISCUSSION TITLE
JDK21 compatible release

### DISCUSSION THREAD

**Discussion by wendigo** [google/error-prone#3967] [Q&A] [answered]
Long story short: latest release isn’t compatible with JDK 21 but the current HEAD SNAPSHOT is. When we could expect new release so we can start using it for JDK 21 compatibility testing?

**Comment by cushon** [comment, marked as answer]
Thanks for the reminder, I'll do some more testing and then try to get a release out this week.

**edit**: 2.20.0 has been released with the fixes for JDK 21 compatibility: https://github.com/google/error-prone/releases/tag/v2.20.0

**Reply by wendigo** [reply]
Great! Thank you :)
